### PR TITLE
PHP8 compatibility updates

### PR DESCRIPTION
**Pull request by markbiek**
## Proposed changes:
These are minor changes as part of our PHP 8.1 compatibility update initiative pdqkMK-UN-p

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:
These are minor enough that I think visual inspection should be sufficient but please let me know if you need more information.

### CODE REVIEW

File: projects/packages/videopress/src/class-stats.php [hunk 1]
```diff
@@ -69,7 +69,7 @@ public static function get_today_plays() {
 
 		$data = $response['data'];
 
-		if ( ! $data || ! isset( $data['days'] ) || count( $data['days'] ) === 0 ) {
+		if ( ! $data || ! isset( $data['days'] ) || ! is_countable( $data['days'] ) || count( $data['days'] ) === 0 ) {
```
Comment (nelsonec87) [inline projects/packages/videopress/src/class-stats.php:72]: I never know when to replace (or not) `count( $var ) === 0` by `$var === array()`

**Review by nelsonec87 — DISMISSED** [review]
LGTM

File: projects/packages/videopress/src/class-stats.php (hunk 1 above)
Comment (markbiek) [reply projects/packages/videopress/src/class-stats.php:72]: Yeah, I'm not sure about that either. Mostly, I've only been changing those when rector flags it.

**Review by nelsonec87 — APPROVED** [review]
LGTM

**Pull request merged**